packages/loader/core-interfaces/src/handles.ts

```diff
@@ -56,6 +56,8 @@ export interface IFluidHandle<
     > extends IProvideFluidHandle {
 
     /**
+     * @deprecated - Use handles instead of absolute paths
```

Comment:
That's not the right comment. It should say something like - Handle's path should not be used for routing. Use `get` to get the underlying object.

Reply:
Also, add this to BREAKING.md

Reply:
Changed it

Reply:
Added note in breaking